### DISCUSSION TITLE
勾选默认设置中文时会同时启用强制unicode字体

### DISCUSSION THREAD

**Discussion by GTEMYee** [Meloong-Git/PCL#546] [提问] [answered]
如题,附图
![ChooseDefaultChinese](https://user-images.githubusercontent.com/117887221/206406620-686c793e-0b07-4cdc-8346-cc9ce9a02f50.jpg)
![MainPage](https://user-images.githubusercontent.com/117887221/206406653-ca8fe69c-f84f-4f69-935d-814a7169db72.png)
![ForceUnicodeEnabled](https://user-images.githubusercontent.com/117887221/206407510-4c43771b-3aa8-4e68-86f8-f52df368df4e.png)
是有意为之还是有其他原因

**Comment by LTCatt** [comment, marked as answer]
这俩是一起设置的……

**Reply by LTCatt** [reply]
你如果觉得这样不行可以交个 issue

**Reply by GTEMYee** [reply]
我只是想知道启用 强制unicode字体 有什么目的或者什么原因
这并不是我需要迫切解决的问题,我不着急交issue

**Reply by LTCatt** [reply]
因为英文下使用像素化字体整个游戏都是像素化字体，文本风格是统一的。
而中文使用像素化字体会导致一部分字体像素化而另一部分不是，文本风格不统一。

**Reply by lim114514** [reply]
我的建议还是默认关闭强制Unicode字体，或者弄一个个性化...
